src/context-items/rename.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { ContextBuilder } from '../context-builder';
+import { Meta } from '../base';
+import { Rename } from './rename';
+
+it('the new path is identical to the old one', () => {
+  const context = new ContextBuilder().setFields(['foo']).build();
+  const meta: Meta = { req: {}, location: 'body', path: 'foo' };
+
+  expect(new Rename('foo').run(context, 'value', meta)).resolves;
```

Comment:
```suggestion
  await expect(new Rename('foo').run(context, 'value', meta)).resolves;
```

though I'm afraid that `.resolves` might not be asserting anything 🤔